src/data/bundle.lean

```diff
@@ -4,21 +4,30 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Nicolò Cavalleri
 -/
 
-import tactic.basic
 import algebra.module.basic
+import data.right_inv
+import data.pi
 
 /-!
 # Bundle
+
 Basic data structure to implement fiber bundles, vector bundles (maybe fibrations?), etc. This file
 should contain all possible results that do not involve any topology.
+
 We provide a type synonym of `Σ x, E x` as `bundle.total_space E`, to be able to endow it with
-a topology which is not the disjoint union topology `sigma.topological_space`. In general, the
-constructions of fiber bundles we will make will be of this form.
+a topology which is not the disjoint union topology. In general, the constructions of fiber bundles
+we will make will be of this form.
 
 ## References
+
 - https://en.wikipedia.org/wiki/Bundle_(mathematics)
+
 -/
 
+/-- TEMPORARY: to be removed when Lean will be updated. -/
+instance lift_fn_range' {α : Sort*} {A : α → Sort*} {B : α → Sort*} [Π i, has_lift_t (A i) (B i)] :
+  has_lift (Π i, A i) (Π i, B i) := ⟨λ f i, ↑(f i)⟩
+
```

Comment:
This now exists as [lift_pi_range](https://leanprover-community.github.io/mathlib_docs/init/coe.html#lift_pi_range), right?

Reply:
Oh yes right I forgot I made the PR to Lean myself haha